### DISCUSSION TITLE
Guidelines for translators

### DISCUSSION THREAD

**Discussion by PrinceEntree8** [geerlingguy/ansible-for-devops-manuscript#2] [Ideas]
Hello @geerlingguy ,
I'm glad you share this project.
I was thinking if we (as community) can make translations of the book.

I think, we can discuss here how to make a translation model/template.

**Comment by delgh1** [comment]
GNU has a tool called GNUN to automate the update of gnu.org, and it can flag out-of-date translations once the English version is updated. It's quite interesting, but I don't yet understand how it works.
In any case, we will need a tool or a script that detects changes in the manuscripts, then remix the translated files with "up2date" paragraphs and "outofdate" paragraphs commented out.
I feel like this would be a major, non-trivial work.
